### PR TITLE
Add 0.5.0 release blog

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

**Description**

Announce Meshery 0.5.0 

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Aisuko** [comment]
Screenshot from 2021-01-30 14-12-04

No permission to check the web site

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM node:alpine3.11
```
Comment (Aisuko) [inline Dockerfile:1]: Always get error due to the internet issue, need someone help to test the Dockerfile can be used

```
npm notice 
npm notice New minor version of npm available! 7.4.3 -> 7.5.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.5.0>
npm notice Run `npm install -g npm@7.5.0` to update!
npm notice 
npm ERR! code ECONNRESET
npm ERR! network aborted
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-01-30T04_14_25_447Z-debug.log
The command '/bin/sh -c npm install' returned a non-zero code: 1
```

File: src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Major themes
+
+The two new adapters with the latest version of Meshery released
+
+* Meshery Adapter for  <a href="https://github.com/layer5io/meshery-traefik-mesh">Traefik Mesh (beta)</a>
+* Meshery Adapter for <a href="https://github.com/layer5io/meshery-nginx-sm">NGINX SM Mesh (beta)</a>
+
+* Dynamic injection of Remote Provider extensions
+  * Need more information
```
Comment (Aisuko) [inline src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx:38]: Need more inforamtion

**Comment by Aisuko** [comment]
Relation to the issues:

https://github.com/layer5io/meshery/issues/2267
https://github.com/layer5io/meshery/issues/2270

File: Dockerfile (hunk 1 above)
Comment (leecalcote) [reply Dockerfile:1]: Are you good to go now, @Aisuko?

File: src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx (hunk 2 above)
Comment (leecalcote) [reply src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx:38]: Here we are - https://docs.meshery.io/extensibility/providers

File: Dockerfile (hunk 1 above)
Comment (Aisuko) [reply Dockerfile:1]: Hard to say, but I always get stuck on those problems.

**Comment by leecalcote** [comment]
Will call for review on this PR in today's community call. Thanks for this, @Aisuko.

**Comment by Aisuko** [comment]
Related to issue https://github.com/layer5io/meshery/issues/2271

**Comment by Aisuko** [comment]
@leecalcote Update the new contents and need more information I guess.

File: src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import { BlogWrapper } from "../Blog.style.js";
+import operator from "./meshery-operator-dark.svg";
+
+<BlogWrapper>
+
+<p style="color:#3c494f;text-align:center;">Thanks to our open source contributors</p>
+
+We're pleased to announce the realease of Meshery 0.5.0, our first release of 2021! This realease consists of number of significant architectural enhancements and 8 new capabilities are entering alpha.
+
+The 0.5.0 release cycle will good begin for 2021 release cycle, this is one of the most feature dense releases in a while:the Meshery innovation
```
Comment (mgfeller) [inline src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx:26]: The 0.5.0 release starts off the 2021 release cycle with a one of the most...

space after  `:`.

File: src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+* Meshery Adapter for <a href="https://github.com/layer5io/meshery-nginx-sm">NGINX SM Mesh (beta)</a>
+  * We support NGINX SM mesh lifecycle management and sample applications operations below
+    * Bookinfo, see the <a href="https://docs.meshery.io/guides/sample-apps#bookinfo">BookInfo Document</a>
+    * HttpBin, see the <a href="https://docs.meshery.io/guides/sample-apps#httpbin">HttpBin Document</a>
+    * Emojivoto, see the <a href="https://docs.meshery.io/guides/sample-apps#emoji.voto">Emojivoto Document</a>
+    * Nginx Servce Mesh Books, see the <a href="https://github.com/BuoyantIO/booksapp">Nginx Servce Mesh Books Document</a>
+
+* Dynamic injection of Remote Provider extensions
+  * Designed to be pluggable
+    * Pluggable UI funcitonality
```
Comment (mgfeller) [inline src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx:48]: functionality

File: src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    * Pluggable Backend funcitonality
```
Comment (mgfeller) [inline src/collections/blog/2021-01-25-announcing-meshery-v0.5/index.mdx:49]: functionality

**Review by mgfeller — COMMENTED** [review]
@Aisuko nice overview :+1:  commented a few minor suggestions and some other details.

File: Dockerfile (hunk 1 above)
Comment (mgfeller) [reply Dockerfile:1]: Same here, Dockerfile doesn't build:

```
 ---> Running in 9bddba93ef3d
npm notice 
npm notice New patch version of npm available! 7.5.1 -> 7.5.2
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.5.2>
npm notice Run `npm install -g npm@7.5.2` to update!
npm notice 
npm ERR! code 1
npm ERR! path /app/node_modules/mozjpeg
npm ERR! command failed
npm ERR! command sh -c node lib/install.js
npm ERR! ⚠ spawn /app/node_modules/mozjpeg/vendor/cjpeg ENOENT
npm ERR!   ⚠ mozjpeg pre-build test failed
npm ERR!   ℹ compiling from source
npm ERR!   ✖ Error: Command failed: /bin/sh -c autoreconf -fiv
npm ERR! /bin/sh: autoreconf: not found
npm ERR! 
npm ERR! 
npm ERR!     at /app/node_modules/bin-build/node_modules/execa/index.js:231:11
npm ERR!     at runMicrotasks (<anonymous>)
npm ERR!     at processTicksAndRejections (node:internal/process/task_queues:94:5)
npm ERR!     at async Promise.all (index 0)

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-06T11_23_00_158Z-debug.log

```

 neither `make setup-libs`:

```
npm ERR! Linux 4.15.0-135-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code EMISSINGARG

npm ERR! typeerror Error: Missing required argument #1
npm ERR! typeerror     at andLogAndFinish (/usr/share/npm/lib/fetch-package-metadata.js:31:3)
npm ERR! typeerror     at fetchPackageMetadata (/usr/share/npm/lib/fetch-package-metadata.js:51:22)
npm ERR! typeerror     at resolveWithNewModule (/usr/share/npm/lib/install/deps.js:456:12)
npm ERR! typeerror     at /usr/share/npm/lib/install/deps.js:457:7
npm ERR! typeerror     at /usr/share/npm/node_modules/iferr/index.js:13:50
npm ERR! typeerror     at /usr/share/npm/lib/fetch-package-metadata.js:37:12
npm ERR! typeerror     at addRequestedAndFinish (/usr/share/npm/lib/fetch-package-metadata.js:82:5)
npm ERR! typeerror     at returnAndAddMetadata (/usr/share/npm/lib/fetch-package-metadata.js:117:7)
npm ERR! typeerror     at pickVersionFromRegistryDocument (/usr/share/npm/lib/fetch-package-metadata.js:134:20)
npm ERR! typeerror     at /usr/share/npm/node_modules/iferr/index.js:13:50
npm ERR! typeerror This is an error with npm itself. Please report this error at:
npm ERR! typeerror     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/michael/dev/layer5/npm-debug.log
Makefile:2: recipe for target 'setup-libs' failed
make: *** [setup-libs] Error 1
```
```

**Comment by Aisuko** [comment]
@mgfeller Thank you, sir. 

I saw some words that have the same spell mistakes on our website some like https://docs.meshery.io/service-meshes/adapters/nginx-sm#sample-applications. Should we add the spell checker on the Doc project? @leecalcote

File: Dockerfile (hunk 1 above)
Comment (Aisuko) [reply Dockerfile:1]: Good, I will check the Dockerfile contents.

**Comment by Aisuko** [comment]
The CI make me confused, you can check at the local environment

```
➜  layer5 git:(feat/release-new) ✗ npm run build

> Layer5@1.0.0 build /home/aisuko/Documents/layer5
> gatsby build

success open and validate gatsby-configs - 0.071s
success load plugins - 0.375s
success onPreInit - 0.050s
success delete html and css files from previous builds - 0.003s
success initialize cache - 0.004s
success copy gatsby files - 0.021s
success onPreBootstrap - 0.010s
success createSchemaCustomization - 0.006s
success Checking for changed pages - 0.001s
success source and transform nodes - 4.472s
success building schema - 0.296s
success create redirects
info Total nodes: 1949, SitePage nodes: 203 (use --verbose for breakdown)
success createPages - 0.141s
success Checking for changed pages - 0.000s
success createPagesStatefully - 0.130s
success Cleaning up stale page-data - 0.001s
success update schema - 0.020s
success onPreExtractQueries - 0.001s
success extract queries from components - 0.477s
success write out redirect data - 0.001s
success Build manifest and related icons - 0.045s
success onPostBootstrap - 0.049s
info bootstrap finished - 8.552s
success run static queries - 9.788s - 18/18 1.84/s
success run page queries - 33.220s - 236/236 7.10/s
success write out requires - 0.005s
success Building production JavaScript and CSS bundles - 36.080s
success Rewriting compilation hashes - 0.002s
success Building HTML renderer - 27.649s
success Building static HTML for pages - 1.628s - 236/236 144.98/s
success Generating image thumbnails - 109.381s - 1464/1464 13.38/s
success onPostBuild - 0.049s
info Done building in 118.03 sec
```

```
➜  layer5 git:(feat/release-new) ✗ npm start                 

> Layer5@1.0.0 start /home/aisuko/Documents/layer5
> npm run develop


> Layer5@1.0.0 develop /home/aisuko/Documents/layer5
> env-cmd -f .env.development gatsby develop

success open and validate gatsby-configs - 0.090s
success load plugins - 0.393s
success onPreInit - 0.031s
success initialize cache - 0.004s
success copy gatsby files - 0.018s
success onPreBootstrap - 0.010s
success createSchemaCustomization - 0.005s
success Checking for changed pages - 0.001s
success source and transform nodes - 1.243s
success building schema - 0.298s
success create redirects
info Total nodes: 1950, SitePage nodes: 203 (use --verbose for breakdown)
success createPages - 0.161s
success Checking for changed pages - 0.001s
success createPagesStatefully - 0.137s
success update schema - 0.019s
success write out redirect data - 0.001s
success Build manifest and related icons - 0.053s
success onPostBootstrap - 0.058s
info bootstrap finished - 4.442s
success onPreExtractQueries - 0.001s
success extract queries from components - 0.512s
success write out requires - 0.059s
success run static queries - 0.562s - 17/17 30.25/s
⠀
You can now view Layer5 in the browser.
⠀
  http://localhost:8000/
⠀
View the GraphQL Playground, an in-browser IDE, to explore your site's data and schema
⠀
  http://localhost:8000/___graphql

```

**Comment by leecalcote** [comment]
@Aisuko, I'm not sure if you have seen these two make commands yet, so I'll point them out just in case:

1. `make setup-libs`
2. `make site`

These essentially do what you've done above.

**Comment by Aisuko** [comment]
> @Aisuko, I'm not sure if you have seen these two make commands yet, so I'll point them out just in case:
> 
>     1. `make setup-libs`
> 
>     2. `make site`
> 
> 
> These essentially do what you've done above.

I saw them, the commands are work on my local but not work well for CI

File: Dockerfile (hunk 1 above)
Comment (mgfeller) [reply Dockerfile]: using `nvm` fixed the errors I got

Comment (Aisuko) [reply Dockerfile]: Ok, I will try nvm

File: Makefile [hunk 6]
```diff
@@ -1,8 +1,20 @@
-setup-libs:
+.PHONY: npm-install
+npm-install:
 	npm install
 
-site:
+.PHONY: npm-start
```
Comment (leecalcote) [inline Makefile:5]: @Aisuko we use `make site` in all 6 of the websites managed by the community.

Comment (Aisuko) [reply Makefile:5]: I will replace the `npm-start`

Comment (Aisuko) [reply Makefile:5]: I did a new PR to the new master branch, Should we work on that one or go on this one?

**Comment by leecalcote** [comment]
Oh, ok. Yes, working on the new one sounds great. 👍

**Comment by Aisuko** [comment]
Close due to https://github.com/layer5io/layer5/pull/1597

**Pull request closed without merging**